designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiHttpServiceBusinessSpec.scala

```diff
@@ -229,6 +228,68 @@ class ScenarioActivityApiHttpServiceBusinessSpec
     }
   }
 
+  "The scenario delete attachment endpoint when" - {
+    "add and delete attachment to existing scenario" in {
+      given()
+        .applicationState {
+          createSavedScenario(exampleScenario)
+        }
+        .when()
+        .basicAuthAllPermUser()
+        .streamBody(fileContent = fileContent, fileName = fileName)
+        .post(s"$nuDesignerHttpAddress/api/processes/$exampleScenarioName/1/activity/attachments")
+        .Then()
+        .statusCode(200)
+
+      val attachmentId = verifyAttachmentAddedActivityExists(
+        user = "allpermuser",
+        scenarioName = exampleScenarioName,
+        fileIdPresent = true,
+        filename = fileName,
+        fileStatus = "AVAILABLE",
+        overrideDisplayableName = fileName
+      ).extractString("activities[1].attachment.file.id")
```

Comment:
maybe `verifyAttachmentAddedActivityExists` could return a wrapper around the currently returned `ValidatableResponse` and encapsulate the string extraction under some method?

Reply:
Name changed to `verifyAttachmentAddedActivityExistsAndReturnFileId`, with return type Option[Long]

Reply:
IMO it'd be better to leave the name as it was before and return the wrapper I mentioned above. When users (of the method) are interested in extracting sth they can do it, but when they are not interested in it, they won't be bothered. 

Moreover, we could easily add other methods to the wrapper that would extract totally different things from the response.

Reply:
done, restored name, created very simple wrapper for extracting fields from activities